src/main/java/com/savvato/tribeapp/entities/ReviewDecisionReason.java

```diff
@@ -0,0 +1,29 @@
+package com.savvato.tribeapp.entities;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name="review_decision_reason")
+public class ReviewDecisionReason {
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    public String getReason() {
+        return reason;
+    }
+
+    public void setReason(String reason) {
+        this.reason = reason;
+    }
+
+    private String reason;
+}
```

Comment:
Add an all-parameter, and no-parameter constructor.

Reply:
These changes have been added in now.